README.md

```diff
@@ -104,6 +113,16 @@ See [HACKING.md](HACKING.md) for further instructions
 
 ## Command interface
 
+### Shell Keys
+- Enter
+ - Runs the statement
```

Comment:
We need one more space to make it a sublist, currently it's rendered as the same level of the keys list

Reply:
good catch. should have verified this before opening the PR